### PR TITLE
[Improvement-4326][ui] Add description field to data source list

### PR DESCRIPTION
**Pull request by xxjingcd**
## What is the purpose of the pull request

 Add description field to data source list. resolve the issue [#4326](https://github.com/apache/incubator-dolphinscheduler/issues/4326)
this close #4362 
## Brief change log

modify dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/list.vue

## Verify this pull request

ui improvement. check datasource  page ui：

- *cd  dolphinscheduler-ui*
- *run "npm run start"*
- *go to "http://localhost:8888/#/datasource/list"*

### CODE REVIEW

File: dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/list.vue [hunk 1]
```diff
@@ -32,6 +32,7 @@
             </div>
           </template>
         </el-table-column>
+        <el-table-column prop="note" :label="$t('Description')"></el-table-column>
```
Comment (zhuangchong) [inline dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/list.vue:35]: <el-table-column :label="$t('Description')" min-width="100">
          <template slot-scope="scope">
            <span>{{scope.row.note | filterNull}}</span>
          </template>
        </el-table-column>

Comment (xxjingcd) [reply dolphinscheduler-ui/src/js/conf/home/pages/datasource/pages/list/_source/list.vue]: ok,I have corrected and checked again

**Review by break60 — APPROVED** [review]
+1

**Comment by zhuangchong** [comment]
+1

**Review by CalvinKirs — APPROVED** [review]
+1

**Comment by CalvinKirs** [comment]
considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(Kris_Evil), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.

**Comment by xxjingcd** [comment]
> considering that it's your first contribution, I think we can get deep communiction, you can contact me through mail or add wechat(Kris_Evil), when mail or added, please tell me who you are, I think I can help to familiar with the DolphinScheduler if you meet with problems.

nice, I have added you to WeChat

**Pull request merged**